src/test/groovy/com/netgrif/application/engine/petrinet/service/PetriNetServiceTest.groovy

```diff
@@ -160,4 +168,51 @@ class PetriNetServiceTest {
         List<PetriNet> petriNets = petriNetService.findAllByUriNodeId(myNode.id)
         assert petriNets.size() == 2
     }
+
+    @Test
+    void processSearch() {
+        long processCount = petriNetRepository.count()
+
+        petriNetService.importPetriNet(stream(NET_SEARCH_FILE), VersionType.MAJOR, superCreator.getLoggedSuper())
+        petriNetService.importPetriNet(stream(NET_FILE), VersionType.MAJOR, superCreator.getLoggedSuper())
```

Comment:
Would be better to use different author for one net to see if search by author works properly

Reply:
done